kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ListConsumerLagsResource.java

```diff
@@ -0,0 +1,113 @@
+import io.confluent.rest.annotations.PerformanceMetric;
+import java.util.Comparator;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.MediaType;
+
+
+@Path("/v3/clusters/{clusterId}/consumer-groups/{consumerGroupId}/lags")
+@ResourceName("api.v3.consumer-lags.*")
+public class ListConsumerLagsResource {
+
+  private final Provider<ConsumerLagManager> consumerLagManager;
+  private final CrnFactory crnFactory;
+  private final UrlFactory urlFactory;
+
+  @Inject
+  public ListConsumerLagsResource(
+      Provider<ConsumerLagManager> consumerLagManager,
+      CrnFactory crnFactory,
+      UrlFactory urlFactory
+  ) {
+    this.consumerLagManager = requireNonNull(consumerLagManager);
+    this.crnFactory = requireNonNull(crnFactory);
+    this.urlFactory = requireNonNull(urlFactory);
+  }
+
+
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @PerformanceMetric("v3.consumer-lags.list")
+  @ResourceName("api.v3.consumer-lags.list")
+  public void listConsumerLags(
+      @Suspended AsyncResponse asyncResponse,
+      @PathParam("clusterId") String clusterId,
+      @PathParam("consumerGroupId") String consumerGroupId
+  ) {
+    CompletableFuture<ListConsumerLagsResponse> response =
+        consumerLagManager.get()
+            .listConsumerLags(clusterId, consumerGroupId)
+            .thenApply(
+                lags -> {
+                  if (lags.isEmpty()) {
+                    throw new NotFoundException("Consumer lags not found.");
+                  }
+                  return lags; })
+            .thenApply(
+                lags ->
+                    ListConsumerLagsResponse.create(
+                        ConsumerLagDataList.builder()
+                            .setMetadata(
+                                ResourceCollection.Metadata.builder()
+                                    .setSelf(
+                                        urlFactory.create(
+                                            "v3",
+                                            "clusters",
+                                            clusterId,
+                                            "consumer-groups",
+                                            consumerGroupId,
+                                            "lags"))
+                                    .build())
+                            .setData(
+                                lags.stream()
+                                    .map(
+                                        lag ->
+                                            GetConsumerLagResource.toConsumerLagData(
+                                                lag, urlFactory, crnFactory))
```

Comment:
calls `GetConsumerLagResource`'s `toConsumerLagData` to avoid having a duplicate of the method in this class